components/ntp_background_images/browser/view_counter_service.cc

```diff
@@ -253,4 +278,28 @@ std::string ViewCounterService::GetSuperReferralCode() const {
   return service_->GetSuperReferralCode();
 }
 
+void ViewCounterService::UpdateP3AValues() const {
+  uint64_t new_tab_count = new_tab_count_state_->GetHighestValueInWeek();
+  constexpr int kNewTabCount[] = {0, 1, 4, 9, 21, 51, 101};
+  const int* it_count =
+      std::lower_bound(kNewTabCount, std::end(kNewTabCount), new_tab_count);
+  int answer = it_count - kNewTabCount;
+  UMA_HISTOGRAM_EXACT_LINEAR("Brave.NTP.NewTabsCreated", answer,
+      7);
+
+  uint64_t branded_new_tab_count =
+      branded_new_tab_count_state_->GetHighestValueInWeek();
+  if (branded_new_tab_count == 0 || new_tab_count == 0) {
+    answer = 0;
+  } else {
+    double ratio = (branded_new_tab_count / (double)new_tab_count) * 100;
+    constexpr double kSponsoredRatio[] = {1.0, 10.0, 20.0, 30.0, 40.0, 50.0};
```

Comment:
the ticket doesn't mention `1.0` 🤔

Reply:
These are the lower bounds, right?
0) 0% => 0
1) > 0% and < 10% => 1
2) 10% to < 20% => 10
3) 20% to < 30% => 20
4) 30% to < 40% => 30
5) 40% to < 50% =>40
6) 50% or more => 50